src/web-ui/src/public/components/DemoGuide.vue

```diff
@@ -0,0 +1,220 @@
+                <p style="margin:1rem;font-weight:bold;font-size:x-large">
+                USE CASES ENABLED IN THIS DEMO
+                </p>
+              <p>Click here to dive deep into the ways Amazon Personalize and other ML Services are used in this demo</p>
+              </div>
+              <div class="demo-guide-use-cases col-sm-12 col-md-3 col-lg-3" v-on:click="showDemoGuidePersonalizeTopics()">
+                <p style="margin:1rem;font-weight:bold;font-size:x-large">
+                AMAZON PERSONALIZE USEFUL TOPICS
+                </p>
+              <p>Dive deeper into some popular topics related to Amazon Personalize</p>
+              </div>
+              <div class="demo-guide-use-cases col-sm-12 col-md-3 col-lg-3">
+                <p style="margin:1rem;font-weight:bold;font-size:x-large">
+                ABOUT THIS DEMO
+                </p>
+              <p>Provide feedback or learn more about the workshops available</p>
+              </div>
+            </li>    
+          </ul>                                    
+      </nav>  
+    </div>
+    <DemoGuideLevel3 :showUseCases="showUseCases" :showPersonalizeTopics="showPersonalizeTopics" :showguidedetails="showguidedetails" @toggleUseCases="hideDemoGuideLevel3()"></DemoGuideLevel3>
+  </div>
+</template>
+
+<script>
+
+import DemoGuideLevel3 from './DemoGuideLevel3.vue';
+
+export default {
+  name: 'DemoGuide',
+  components: {
+    DemoGuideLevel3
+  },
+  props: {
+  },
+  data () {
+    return {  
+      errors: [],
+      showguide: false,
+      showguidedetails: false,
+      sectionName: 'Amazon Personalize',
+      displaySection: 'User Personalization',
+      showPersonalizeTopics: false,
+      showUseCases: false,
+      services: [{
+        name: 'Amazon Personalize',
+        subsections: [
+          {name : 'User Personalization'},
+          {name : 'Similar items'},
+          {name : 'Personalized ranking'},
+          {name : 'Real time events'}
+          ]
+        },
+        {
+          name: 'Amazon Pinpoint',
+          subsections: [
+            {name : 'Welcome Email'},
+            {name : 'Abandoned cart email'},
+            {name : 'Text alerts(SMS)'}
+          ]
+        },
+        {
+          name: 'Amazon Lex',
+          subsections: [
+            {name : 'Chatbot conversational interfaces'},
+          ]
+        }
+      ]
+    }
+  },
+  computed: {
+    showLevel: function() {
+      return this.showguide || this.showguidedetails;
+    }
+  },
+  methods: {
+    showDemoGuideLevel2: function() {
+      this.showguide = !this.showguide;
+      this.showUseCases = false;
+      this.showPersonalizeTopics = false;
+    },
+    showDemoGuideLevel3: function() {
+      this.showguide = !this.showguide;
+      this.showguidedetails = !this.showguidedetails;
+    },
+    hideDemoGuideLevel3: function() {
+      this.showguide = !this.showguide;
+      this.showguidedetails = !this.showguidedetails;
```

Comment:
I see some members following camel case (eg: showUseCases) and some members that don't (eg: showguidedetails)
Is there a reason not to follow the same pattern?

Reply:
No reason. I can fix it.